client/src/modules/lead/EditLead/EditLeadContent/EditLeadHistory/Notes/noteReducer.ts

```diff
@@ -0,0 +1,32 @@
+import Note from '../../../../../../models/Note';
+import { CREATE_NOTE, DELETE_NOTE, LOAD_NOTES, UPDATE_NOTE } from './types';
+
+interface Action {
+  type: string;
+  payload: any;
+}
+
+const initialState: Note[] = [];
+
+export default function(state = initialState, action: Action) {
```

Comment:
if our state is only consist of array of notes, better use `notes` then `state` and `initialNotes` instead of `initialState`, its a bit easier to understanding